schedule/staging/sles4sap_offline_gnome@64bit-staging.yaml

```diff
@@ -15,7 +15,12 @@ vars:
   SLES4SAP_MODE: sles4sap
   SLE_PRODUCT: sles4sap
   SYSTEM_ROLE: default
-  WORKAROUND_MODULES: base,desktop,sapapp,serverapp,ha
+  ADDONURL: base,desktop,sapapp,serverapp,ha
+  ADDONURL_BASE: ftp://openqa.suse.de/%REPO_SLE_MODULE_BASESYSTEM%
```

Comment:
I don't remember we have this implemented in yaml parser. You will need to put it in the test suites.

Reply:
Variable expansion works as those are vars.json variables.